docs/Manifest.toml

```diff
@@ -0,0 +1,80 @@
+# This file is machine-generated - editing it directly is not advised
+
+[[Base64]]
+uuid = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
+
+[[Dates]]
+deps = ["Printf"]
+uuid = "ade2ca70-3891-5945-98fb-dc099432e06a"
+
+[[Distributed]]
+deps = ["Random", "Serialization", "Sockets"]
+uuid = "8ba89e20-285c-5b6f-9357-94700520ee1b"
+
+[[DocStringExtensions]]
+deps = ["LibGit2", "Markdown", "Pkg", "Test"]
+git-tree-sha1 = "1df01539a1c952cef21f2d2d1c092c2bcf0177d7"
+uuid = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+version = "0.6.0"
+
+[[Documenter]]
```

Comment:
I don't think you need to add the documenter to the package dependencies. You can call `Pkg.add` when calling the Documentation Stage

Reply:
This is a separate Project.toml just for docs. So it's not part of package dependencies. There are other documentation package extensions (see [DocStringExtensions.jl](https://github.com/JuliaDocs/DocStringExtensions.jl)) that I'd like to add in the future and having a separate Project.toml for this can be useful.